test/fmw/gtest/include/gtest/internal/gtest-port.h

```diff
@@ -263,8 +263,8 @@
 # define GTEST_OS_HPUX 1
 #elif defined __native_client__
 # define GTEST_OS_NACL 1
-#elif defined __OpenBSD__
-# define GTEST_OS_OPENBSD 1
+#elif defined(__OpenBSD__) || defined(__FreeBSD__) || defined(__OpenBSD__)
```

Comment:
Looks like one of these OpenBSDs should read NetBSD

Reply:
I pushed a change to fix this.

Reply:
Thanks Greg.